### ISSUE TITLE
User name not recognised

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#1232]
Creating data file (add data file from folder page) in "share with a person" pop up window user name is not recognised if typing e.g. "Stuart" or "olga"

**Comment by OlgaKrebs** [comment]
![sharingPerson](https://user-images.githubusercontent.com/55085007/199701503-840586c9-296c-44eb-b1c4-0ae5b08b50ea.jpg)

**Comment by OlgaKrebs** [comment]
by creating new study person recognising works correctly

**Comment by OlgaKrebs** [comment]
same issue
 if I try to add new Creator for data file  mentioned above 
![creatorPerson](https://user-images.githubusercontent.com/55085007/199702832-9c6136d8-cb99-4af5-a1a7-f0e51ad85612.jpg)

**Comment by stuzart** [comment]
I'm unable to reproduce this. From the screenshot, are you trying to do it through the single page view? It could be an issue with the way the form is embedded.

**Comment by stuzart** [comment]
I can confirm it is specific to the Single Page View, it works in other places

**Comment by OlgaKrebs** [comment]
Yes, Stuart - as i mentioned- in other places it works, but if "adding file to folder" from single page view it doesn't - reproduced it many times

**Comment by floradanna** [comment]
@vdkkia could you please take a look at this issue, when you can?

**Comment by vdkkia** [comment]
I tried to reproduce it locally and on the dataHub Test and Development servers. It seems it's working.
